Example/HCaptcha_Tests/mock.html

```diff
@@ -1,11 +1,25 @@
 <html>
   <head>
     <meta name="viewport" content="width=device-width" />
+    <script type="text/javascript">
+      // onerror need to report errors which happens on WKWebView.loadHTMLString
+      window.onerror = function(msg, url, line, column, error) {
+        if (window.webkit) {
+          window.webkit.messageHandlers.hcaptcha.postMessage({ "error": 1 });
+        } else {
+          console.log("Error:", error);
+        }
+      };
+    </script>
```

Comment:
I just found that there is no way to catch errors on  `WKWebView.loadHTMLString` without `window.onerror`.
@e271828- how do you think should we add a similar approach to our main HTML template `hcaptcha.html`?

Reply:
https://stackoverflow.com/a/61031417 is the only pattern I'm aware of that reliably works for this.

Reply:
@e271828- we use absolutely the same approach (according to our error handling approach)

Reply:
Yep, I'm aware :)